### PR TITLE
Update node-module-in-edge-runtime.md

### PR DESCRIPTION
**Pull request by leerob**
To be a bit more clear.

### CODE REVIEW

File: errors/node-module-in-edge-runtime.md [hunk 1]
```diff
@@ -1,39 +1,25 @@
...
-export default async function middleware() {
-  if (process.version) {
-    spawn('ls', ['-lh'])
-  }
-  return NextResponse.next()
-}
-```
-
-Dynamic imports in unreachable code path is also supported.
+> **Note:** You can import Node.js modules as long as they are not used.
```
Comment (styfle) [inline errors/node-module-in-edge-runtime.md:15]: Do we need to clarify why you would want to import something unused?

File: errors/node-module-in-edge-runtime.md [hunk 2]
```diff
@@ -1,39 +1,25 @@
...
 
-- [Edge runtime supported APIs and primitives][edge-primitives]
+- [Edge Runtime Supported APIs][edge-primitives]
 - [Next.js Middleware][middleware]
 
 [middleware]: https://nextjs.org/docs/advanced-features/middleware
 [routes]: https://nextjs.org/docs/api-routes/edge-api-routes
 [node-primitives]: https://nodejs.org/api/index.html
-[edge-primitives]: https://edge-runtime.vercel.app/features/available-apis
+[edge-primitives]: https://nextjs.org/docs/api-reference/edge-runtime#unsupported-apis
```
Comment (styfle) [inline errors/node-module-in-edge-runtime.md:25]:
```suggestion
[unsupported-apis]: https://nextjs.org/docs/api-reference/edge-runtime#unsupported-apis
```

I think the link name need to be updated too

File: errors/node-module-in-edge-runtime.md (hunk 1 above)
Comment (leerob) [reply errors/node-module-in-edge-runtime.md]: Nope, removed.

File: errors/node-module-in-edge-runtime.md (hunk 2 above)
Comment (leerob) [reply errors/node-module-in-edge-runtime.md]: Fixed.

**Review by styfle — DISMISSED** [review]

**Review by ijjk — DISMISSED** [review]

**Review by ijjk — APPROVED** [review]

**Pull request merged**